controller/array_action/utils.py

```diff
@@ -1,3 +1,10 @@
+from controller.array_action.config import UTF_8
+
+
+def bytes_to_string(input_as_bytes):
+    return input_as_bytes.decode(UTF_8) if input_as_bytes else ""

```

Comment:
I added to handle None value which probably may not be. But when I use this method I assume it never raises exception
This is called "defensive programming"

Reply:
I understand that this could protect against future `None`.
but as I wrote, since it is only used in SVC (and only in one place), I think it's too soon to put it in common utils

Reply:
I do not agree. Leaving this way

Reply:
do you know if any other storage type does this conversion?
or you just think we should put storage-specific logic here?

Reply:
This func is not storage specific. It is general purpose func
I am stopping this discussion

Reply:
I think we can avoid all this raw bytes conversion by using `self.client.transport.send_command` (instead of `self.client.send_raw_command`, similar to [the capacity type conversion avoidance trick in ds8k](https://github.com/IBM/ibm-block-csi-driver/pull/122#discussion_r381554222))
so I'm unresolving this for CSI-1492

Reply:
fixed in https://github.com/IBM/ibm-block-csi-driver/pull/432